virttest/libvirt_xml/backup_xml.py

```diff
@@ -116,11 +115,96 @@ def __init__(self, virsh_instance=base.virsh):
                                    tag_name='disk', attribute='backupmode')
             accessors.XMLAttribute('incremental', self, parent_xpath='/',
                                    tag_name='disk', attribute='incremental')
-            accessors.XMLElementDict('target', self, parent_xpath='/',
-                                     tag_name='target')
+            accessors.XMLElementNest('target', self, parent_xpath='/',
+                                     tag_name='target',
+                                     subclass=self.DiskTarget,
+                                     subclass_dargs={
+                                         'virsh_instance': virsh_instance})
             accessors.XMLElementDict('driver', self, parent_xpath='/',
                                      tag_name='driver')
-            accessors.XMLElementDict('scratch', self, parent_xpath='/',
-                                     tag_name='scratch')
+            accessors.XMLElementNest('scratch', self, parent_xpath='/',
+                                     tag_name='scratch',
+                                     subclass=self.DiskScratch,
+                                     subclass_dargs={
+                                         'virsh_instance': virsh_instance})
+
             super(self.__class__, self).__init__(virsh_instance=virsh_instance)
             self.xml = "<disk/>"
+
+        class DiskTarget(base.LibvirtXMLBase):
+            """
+            Backup disk target XML class
```

Comment:
suggest add example what DiskTarget looks like

Reply:
thx for the suggestion, just added two xml samples in class BackupXML's docstring, this may be more clear to instruct the usage. pls have a check.